### PR TITLE
Update google drive gem

### PR DESCRIPTION
**Pull request by domenickpowers**
### Summary

Updated the google_drive gem to version 2 and updated tests

### Other Information

If there's anything else that's important and relevant to your pull
request, mention that information here. This could include
benchmarks, or other information.

Thanks for contributing to Roo!

### CODE REVIEW

File: spec/lib/roo/google_spec.rb [hunk 1]
```diff
@@ -75,9 +75,9 @@ def key_of(spreadsheetname)
     end
     let(:key) { key_of('numbers1') }
     it 'check date fields' do
-      expect(subject.celltype(5,1)).to eq(:date)
-      expect(subject.cell(5,1)).to eq(Date.new(1961, 11, 21))
-      expect(subject.cell(5,1).to_s).to eq('1961-11-21')
+      expect(subject.celltype(5,'A')).to eq(:date)
+      expect(subject.cell(5,'A')).to eq(Date.new(1961, 11, 21))
+      expect(subject.cell(5,'A').to_s).to eq('1961-11-21')
```
Comment (chopraanmol1) [inline spec/lib/roo/google_spec.rb]: Is the above changeset necessary?

File: spec/lib/roo/google_spec.rb [hunk 2]
```diff
@@ -86,7 +86,7 @@ def key_of(spreadsheetname)
     it 'check time fields' do
       # 13:46:12
```
Comment (chopraanmol1) [inline spec/lib/roo/google_spec.rb]: I think this comment should be updated to ` 1:46:12`

**Review by chopraanmol1 — CHANGES_REQUESTED** [review]
Thank you for submitting this PR. It is a huge help.

File: spec/lib/roo/google_spec.rb (hunk 1 above)
Comment (domenickpowers) [reply spec/lib/roo/google_spec.rb]: No probably not, as I was experimenting. It is changed back

**Comment by domenickpowers** [comment]
Updated code based on PR suggestions

**Review by chopraanmol1 — APPROVED** [review]
LGTM. Squash your changes (last 2 commits) into 1 commit.

**Pull request merged**